src/components/Buttons/DeleteSequenceButton.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { Button } from '@dhis2/ui'
+import i18n from '@dhis2/d2-i18n'
+import { DeleteSequenceModal } from '../Modal'
+
+const DeleteSequenceButton = ({ name, onSuccess }) => {
```

Comment:
this is almost the same as the DeleteFormButton component, so I wonder if it would be useful to just reuse the existing component with some adjustments (up to you)

Reply:
Yeah. Currently I think we have the delete sequence/job buttons in the list view, and the delete buttons in the edit views. The modal is at the bottom of the hierarchy and includes the hook for the request.

We could have a button that shows a component on click, with the component being a modal. That seems to be the generic bit of functionality here. I'll create a follow up issue for it.

Reply:
https://dhis2.atlassian.net/browse/DHIS2-15677